scripts/initialize-build-host.sh

```diff
@@ -48,6 +48,13 @@ then
     sed -i -e 's/^\( *Defaults *requiretty *\)$/# \1/' /etc/sudoers
 fi
 
+# Add 512MB swap space, java needs it, and jenkins stops complaining.
+# This will happen on both hosts, if there is a proxy and a build host.
+dd if=/dev/zero of=/swapfile bs=1M count=512
```

Comment:
It is. :-P I messed this up basically, but I pushed new fixes that removed this. Sorry, not my best work, but working with Jenkins spawning is pretty tiresome. :-)